packages/translations/locales/fr/dashboard.json

```
@@ -545,7 +642,17 @@
     "productMarketplace": "Marché des produits",
     "productSuppliers": "Fournisseurs de produits",
     "products": "Produits",
+    "findCandidates": "Trouver des candidats",
+    "findReplacements": "Trouver du personnel de remplacement",
+    "internPool": "Vivier de stagiaires",
```

Biome (2.4.15)

[error] The key internPool was already declared.

(lint/suspicious/noDuplicateObjectKeys)